### PR TITLE
Grammar tweaks

### PR DESCRIPTION
**Pull request by mattlibera**
Hello Dries!

I thought I might provide some more nit-picky grammatical tweaks for this documentation while it is still young! All edits should be cosmetic, and should thus not change the content in any way.

If you are interested in these changes but need me to resubmit a better squashed version, let me know.

Thanks for all you do!
-- Matt

### CODE REVIEW

**Comment by driesvints** [comment]
First of all: I'm sorry for taking this long to review this!

Thank you so much for this PR. Really appreciate all the work you've put into this one 👏

**Comment by mattlibera** [comment]
Hi Dries! Not a problem at all. I'm on the Discord server and see how busy the last few weeks have been for you! Happy to help out in any way I can 👍

File: avatar.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ And this will force unavatar to search only Gravatar images.
 
 ## Fallbacks
 
-If you'd like to provide a fallback image when not matches were found for a given search result you can use the `fallback` attribute:
+If you'd like to provide a fallback image when no matche is found for a given search result, you can use the `fallback` attribute:
```
Comment (rihardssceredins) [inline avatar.md:43]: It should be “match” right?

Comment (driesvints) [reply avatar.md:43]: Fixed. Thanks!

Comment (mattlibera) [reply avatar.md:43]: 🤦 sorry about that one!

**Pull request merged**